src/cloudevent/mocks/database/database-on-value-created.ts

```diff
@@ -0,0 +1,15 @@
+import { MockCloudEventAbstractFactory } from '../../types';
```

Comment:
nit: I started this with `storage` the last time I edited the codebase, but I think that this habit of having a file for every event type is probably overkill. In the CLI at least we have a map of event type to service and then the strategy object is per service.

Reply:
Thats fair. Theres definitely DRYness that could get applied here, but I also wanted to push to make the code as easy-to-read and easy to add.

Adding a new event is a matter of creating a new AbstractFactory with `generateMock` and `match` implemented.

TBH I could go either way.

Reply:
I'll follow up in a future cl.